### PR TITLE
clean up BundleTransport change to LightDark

### PR DESCRIPTION
**Pull request by breed**

### CODE REVIEW

File: BundleTransport/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,34 +2,16 @@
...
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE_DATA_SYNC" />
     <uses-permission android:name="Manifest.permission.POST_NOTIFICATIONS"/>
-    <uses-permission
-        android:name="android.permission.WRITE_EXTERNAL_STORAGE"
-        tools:ignore="ScopedStorage" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-    <uses-permission
-        android:name="android.permission.MANAGE_EXTERNAL_STORAGE"
-        tools:ignore="ScopedStorage" />
-    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (pankinkun) [inline BundleTransport/app/src/main/AndroidManifest.xml:17]: oh do we not need those permissions for wifi direct? or do you plan to move them to android-core?

**Review by pankinkun — APPROVED** [review]

File: BundleTransport/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (breed) [reply BundleTransport/app/src/main/AndroidManifest.xml:17]: i moved them to android core in the last change, although now that you mention it, the one in android-core doesnt' include ScopedStorage? i wonder if we need that? the IDE says that the permission is depricated.

Comment (pankinkun) [reply BundleTransport/app/src/main/AndroidManifest.xml:17]: do you mean Write and Read external storage? I don't think we use it in transport but we have to have permission to external storage in client for the USB

Comment (breed) [reply BundleTransport/app/src/main/AndroidManifest.xml:17]: i used the permissions in the client to populate the permissions in android-core, so we should be covered.

**Pull request merged**